src/templates/profile/body/userInfo/UserInfo.js

```diff
@@ -1,40 +1,220 @@
+import { useEffect, useState } from 'react';
 import { Row, Col } from 'react-bootstrap';
+import UserService from 'keycloak/Keycloak';
 
+/* Import Components */
+import CustomEditIcon from 'templates/general/icons/CustomEditIcon';
+
+/* Import API */
+import PatchUser from 'api/user/PatchUser';
+import GetOrganizations from 'api/organization/GetOrganizations';
+
+
+const UserInfo = (props) => {
+    const userProfile = props.userProfile;
+
+    const [organizations, setOrganizations] = useState([]);
+
+    useEffect(() => {
+        GetOrganizations(Process);
+
+        function Process(organizations) {
+            setOrganizations(organizations);
+        }
+    }, [])
+
+    const [editMode, setEditMode] = useState(false);
+
+    /* Form handling */
+    const [form, setForm] = useState({
+        firstName: userProfile['firstName'],
+        lastName: userProfile['lastName'],
+        email: userProfile['email'],
+        orcid: '',
```

Comment:
Is this because of the API issue?

Reply:
No, forgot to add it, should now display the orcid when the API correctly returns it